### PR TITLE
README.md: rephrase Buildah description

### PR DESCRIPTION
**Pull request by vrothberg**
Rephrase the description of Buildah to make some core attributes
(e.g., rootless, Dockerfile-less build, etc.) more obvious.

Addresses: https://github.com/containers/buildah/issues/1349#issuecomment-466096550
Signed-off-by: Valentin Rothberg <rothberg@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by vrothberg** [comment]
@TomSweeneyRedHat PTAL

**Comment by TomSweeneyRedHat** [comment]
Just a note that the final changes should be made on the libpod README.md too.

File: README.md [hunk 1]
```diff
@@ -35,18 +35,21 @@ For blogs, release announcements and more, please checkout the [buildah.io](http
...
-Podman [here](https://github.com/containers/libpod).  Both Buildah and Podman are
-command line tools that work on OCI images and containers.  The two projects
-differentiate in their specialization.
-
-Buildah specializes in building OCI images.  Buildah's commands replicate all
-of the commands that are found in a Dockerfile. Buildah’s goal is also to
-provide a lower level coreutils interface to build images, allowing people to build
-containers without requiring a Dockerfile.  The intent with Buildah is to allow other
-scripting languages to build container images, without requiring a daemon.
+Buildah and Podman are two complementary open-source projects that are
```
Comment (TomSweeneyRedHat) [inline README.md]: IDK, if it's 100% right, but I converted to "Open-source" based on the wikipedia reference.  https://en.wikipedia.org/wiki/Open-source_software

File: README.md [hunk 2]
```diff
@@ -35,18 +35,21 @@ For blogs, release announcements and more, please checkout the [buildah.io](http
...
+command line tools that work on Open Container Initiative (OCI) images images
+and containers.  The two projects differentiate in their specialization.
+
+Buildah specializes in building OCI images.  It allows building images with and
+without Dockerfiles while not requiring any root privileges.  Buildah’s
+ultimate goal is to provide a lower-level coreutils interface to build images.
+The flexibility of building images without Dockerfiles allows to integrate
+other scripting languages into the build process.  Buildah follows a simple
+fork-exec model and does not run as a daemon but it is based on a comprehensive
+API in golang, which can be vendored into other tools.
```
Comment (TomSweeneyRedHat) [inline README.md]: The point that Buildah's commands replicate the commands found in a Dockerfile has been kind of muted here.  I'd like to keep that as a point of emphasis.  Maybe:  "... OCI images.  It allows building ..." to "... OCI images.  Buildah's commands replicate all of the commands that are found in a Dockerfile.  This allows Buildah to build ... "

Comment (TomSweeneyRedHat) [reply README.md]: Just in case, I mistagged the line, "muted" comment is in regards to the end of the first sentence and the start of the second.

File: README.md [hunk 3]
```diff
@@ -35,18 +35,21 @@ For blogs, release announcements and more, please checkout the [buildah.io](http
...
+[GitHub.com](https://github.com) with Buildah
+[here](https://github.com/containers/buildah) and Podman
+[here](https://github.com/containers/libpod).  Both, Buildah and Podman are
+command line tools that work on Open Container Initiative (OCI) images images
+and containers.  The two projects differentiate in their specialization.
+
+Buildah specializes in building OCI images.  It allows building images with and
+without Dockerfiles while not requiring any root privileges.  Buildah’s
+ultimate goal is to provide a lower-level coreutils interface to build images.
+The flexibility of building images without Dockerfiles allows to integrate
```
Comment (TomSweeneyRedHat) [inline README.md]: "allows to integrate" to "allows for the integration of"

File: README.md [hunk 4]
```diff
@@ -55,12 +58,12 @@ created from those images.
...
 allows users to create "traditional containers" where the intent of these containers is
 to be long lived.  While Buildah containers are really just created to allow content
-to be added back to the container image.   An easy way to think of it is the
+to be added back to the container image.  An easy way to think of it is the
 `buildah run` command emulates the RUN command in a Dockerfile while the `podman run`
 command emulates the `docker run` command in functionality.  Because of this and their underlying
 storage differences, you can not see Podman containers from within Buildah or vice versa.
 
-In short Buildah is an efficient way to create OCI images  while Podman allows
+In short, Buildah is an efficient way to create OCI images while Podman allows
```
Comment (TomSweeneyRedHat) [inline README.md]: oops, missed that earlier, ty!

File: README.md (hunk 3 above)
Comment (vrothberg) [reply README.md]: Thanks!

File: README.md (hunk 2 above)
Comment (vrothberg) [reply README.md]: Thanks for the comment. I tried to address it, can you take another look?

**Comment by TomSweeneyRedHat** [comment]
LGTM, assuming happy tests.
Thanks @vrothberg !

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by vrothberg** [comment]
@rh-atomic-bot r=@rhatdan
